config.yaml

```diff
@@ -99,7 +107,7 @@ params:
       - icon: fa-envelope-o
         title: Mailing list
         description: >-
-          Join PythonPune mailing group to get important announcements and
+          Join **PythonPune mailing group** to get important announcements and
```

Comment:
We should either write mailing list or Google group, but I think mailing list will be better as we use it as mailing list only.